msg/FixedWingLateralSetpoint.msg

```diff
@@ -6,4 +6,4 @@ float32 course 				# [-pi, pi] Course over ground setpoint, w.r.t. North. NAN if
 float32 airspeed_direction    		# [-pi, pi] Angle projected to ground of desired airspeed vector, w.r.t. North. NAN if not controlled directly, used as feedforward if course setpoint is finite.
 float32 lateral_acceleration 		# [m/s^2] Lateral acceleration setpoint in FRD frame. NAN if not controlled directly, used as feedforward if either course setpoint or airspeed_direction is finite.
 
-# TOPICS fixed_wing_lateral_setpoint fixed_wing_lateral_setpoint_status
+# TOPICS fixed_wing_lateral_setpoint
```

Comment:
Don't you get this for free from the name? I mean you needed the TOPICS definition when there were two, but you probably don't need it now, given it matches the name of the message.

Reply:
Yes correct, very well spotted!

Reply:
addressed